webapp/channels/src/entry.tsx

```diff
@@ -59,22 +60,18 @@ function renderRootComponent() {
 }
 
 /**
- * Adds a function to be invoked onload appended to any existing onload
- * event handlers.
+ * Adds a function to be invoked when the DOM content is loaded.
  */
-function appendOnLoadEvent(fn: (evt: Event) => void) {
-    if (window.onload) {
-        const curronload = window.onload;
-        window.onload = (evt) => {
-            (curronload as any)(evt);
-            fn(evt);
-        };
+function appendOnDOMContentLoadedEvent(fn: (evt: Event) => void) {
```

Comment:
Do we need the `evt` argument or can the function just be a `() => void`? We don't use the event anywhere, and we currently make a fake event below which feels odd, so can we just get rid of that parameter?

Reply:
Yes i dont think we need param and i also removed creating a new Event